app/models/role.rb

```diff
@@ -0,0 +1,6 @@
+class Role < ApplicationRecord
+  has_many :person_roles
+  has_many :people, through: :person_roles
+
+  enum name: [:admin]
```

Comment:
Enums are amazing! 

I will caution always using the Hash syntax for them: 
```ruby 
enum name: { 
  admin: 0
}
```
Why do you think that is "safer" than using the array syntax? 

Enums also do amazing things like automatically adding scopes and predicate methods for all items in the enum.

So you can now call `Role.admin`, and get a list of all of the roles that have the admin roles. You can also call `role.admin?` to find out if a role is a particular role.

The above methods might be useful for the Person#admin? method